src/main/java/ay2122s1_cs2103t_w16_2/btbb/model/booking/UniqueBookingList.java

```diff
@@ -24,6 +25,51 @@ public void add(Booking toAdd) {
         internalList.add(toAdd);
     }
 
+    /**
+     * Returns true if the list contains an equivalent booking as the given argument.
+     *
+     * @param toCheck The booking which is used to check.
+     * @return True if the list contains an equivalent booking as the given argument. False otherwise.
+     */
+    public boolean contains(Booking toCheck) {
+        requireNonNull(toCheck);
+        return internalList.stream().anyMatch(toCheck::equals);
+    }
```

Comment:
In clients, the contains() method is used to check for a client with the same identity (phone number) They use isSameClient
For bookings, we have not defined isSameBooking yet. It will likely be different from the equals method. Because the equals method will also check the is_completed field. Whereas the identity of a booking is probably just same phone, same date and same time slot.
Should we not use this yet in case of confusion?